### DISCUSSION TITLE
OIDCProviderSignedJwksUri and mandatory exp claim

### DISCUSSION THREAD

**Discussion by psteniusubi** [OpenIDC/mod_auth_openidc#1182] [General]
Hi Hans & co,
We have run into an interop issue with Signed JWKS: in the mod_auth_openidc implementation the "exp" claim is mandatory and must always be present.
The specification for [Signed JWKS](https://openid.net/specs/openid-federation-1_0-32.html#section-5.2.1) in OpenID Federation defines the "exp" claim is optional.
I have implemented a [fix that resolves](https://github.com/psteniusubi/mod_auth_openidc/commit/5cae798e770a0ae815130d5fabc3351754c268ac) the interop issue. The exp claim is still validated if it's present. I can create a PR if you agree.
Petteri

**Comment by zandbelt** [comment]
thanks for spotting this, please create the PR and I'll merge it before the next release
